README.md

```diff
@@ -20,19 +20,35 @@ https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe-support-script
 
 ### Install the `healthcheck_lite` module
 
-To install the `healthcheck_lite` module, execute the following command on the Primary Master.
+#### Using Bolt
+
+Bolt is the preffered installation and execution method and can be used either on the master or another machine with connectivity to the master. Firstly, follow the instruction for [installing Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html). PE customers are *strongly* encouraged to follow the note about directly installing the package if installing on a PE node.
```

Comment:
`preffered ` is mispelled. Please check all files for spelling errors.

Bolt being preferred is something the team should validate.

This is a PE-specific tool, but `PE customers are ... if installing on a PE node` implies otherwise. In any case, "user" is preferable to "customer" as it is less transactional.

Executing on another machine with connectivity to the master results in the file being on another machine. The output of the `collect` task and this README should account for that.

Reply:
`vim` has spellcheck, how could I forget...

Considering they're officially Bolt Tasks™ now and that Bolt has a lot of advantages, I think it's clear that Bolt is preferred.  We can change it when we finalize this, but my suggestion is to have this language.

Reply:
Installing and configuring Bolt adds friction to the process, and while it is valuable to encourage the use of Bolt, I don't see a benefit for the user or for the support engineer within the context of the Heath Check itself. For example:

> Firstly, follow the instruction for installing Bolt. 

Waves away the details of that installing and configuration.

> PE customers are strongly encouraged to follow the note about directly installing the package.

Waves hands at that risk of package repo conflict.